files/en-us/web/api/webxr_device_api/fundamentals/index.html

```diff
@@ -49,13 +49,13 @@ <h3 id="Field_of_view">Field of view</h3>
 
 <h4 id="What_is_field_of_view">What is field of view?</h4>
 
-<p>The field of view is, at its essence, the extent of the environment you are able to see at any one time. The width of the field of view is measured as the angle defining the arc from the far left edge of your field of view to the far right edge, and is specified in either degrees or radians.</p>
+<p>The field of view is the extent to which you are able to see the environment. The width of the field of view, specified in either degrees or radians, is measured as the angle defining the arc from the far left edge of your field of view to the far right edge.</p>
 
-<p>A human eye is typically able to take in a FOV of around 135°. Assuming a person has two healthy eyes, their total field of view is about 200° to 220° wide. Why is the FOV wider with two eyes, but not double the single-eye FOV? It's because the two eyes FOVs overlap a lot. That overlap is what gives us depth perception; within the overlap area — which is around 115° across — we have depth perception. Outside that overlap area, our vision is monocular and lacks good depth perception.</p>
+<p>A human eye is able to take in a FOV of around 135°. Assuming a person has two healthy eyes, the total field of view ramps up to about 200° to 220° wide. Why is the FOV wider with two eyes, but not double the single-eye FOV? It's because the two eyes' FOVs overlap a lot. That overlap gives us depth perception which is around 115° wide. Outside the overlap area, our vision falls back to monocular.</p>
```

Comment:
```suggestion
<p>A human eye is able to take in a FOV of around 135°. Assuming a person has two healthy eyes, the total field of view ends up being about 200° to 220° wide. Why is the FOV wider with two eyes, but not double the single-eye FOV? It's because the two eyes' FOVs overlap a lot. That overlap gives us depth perception, which is around 115° wide. Outside the overlap area, our vision falls back to monocular.</p>
```
*ramps up to* here doesn’t strike me a particularly idiomatic. It sorta makes it seem a lot more exciting than it actually is. Please consider my suggestion, or some other mundane wording.

Reply:
"ends up" might be a better fit for technical documentation content. Perhaps I was getting to excited abou WebXR :)